lib/pages/search/search.dart

```diff
@@ -225,15 +225,19 @@ class SearchPage extends HookConsumerWidget {
                   ],
                 ),
                 Expanded(
-                  child: AnimatedSwitcher(
-                    duration: const Duration(milliseconds: 300),
-                    child: switch (selectedChip.value) {
-                      "tracks" => const SearchPageTracksTab(),
-                      "albums" => const SearchPageAlbumsTab(),
-                      "artists" => const SearchPageArtistsTab(),
-                      "playlists" => const SearchPagePlaylistsTab(),
-                      _ => const SearchPageAllTab(),
-                    },
+                  child: GestureDetector(
+                    onTap: () => FocusScope.of(context).unfocus(),
```

Comment:
Seems like this is not the correct way to lose focus in modern Flutter:

https://stackoverflow.com/questions/44991968/how-can-i-dismiss-the-on-screen-keyboard